tests/end2end/features/test_prompts_bdd.py

```diff
@@ -37,6 +39,20 @@ def wait_for_prompt(quteproc):
     quteproc.wait_for(message='Asking question *')
 
 
+@bdd.given("I may need a fresh instance")
+def fresh_instance(quteproc):
+    """Restart qutebrowser to bypass webengine's permission persistance."""
+    # Qt6.8 by default will remember feature grants or denies. When we are
+    # on PyQt6.8 we disable that with the new API, otherwise restart the
+    # browser to make it forget previous prompts.
+    if (
+        qtutils.version_check("6.8", compiled=False)
+        and version.PYQT_WEBENGINE_VERSION_STR < "6.8.0"
```

Comment:
Comparing those as a string sounds like a footgun to me... There's `PYQT_WEBENGINE_VERSION` that gives you a hex number:

```python
>>> from PyQt6.QtWebEngineCore import PYQT_WEBENGINE_VERSION
>>> hex(PYQT_WEBENGINE_VERSION)
'0x60700'
```

or I suppose you could instead do a `hasattr()` check, or set a class variable in `WebEngineSettings` depending on whether the setting actually worked or not?